apps/platform/src/app/auth/page.tsx

```diff
@@ -51,6 +52,49 @@ export default function AuthPage(): React.JSX.Element {
     }
   }
 
+  const handleGithubLogin = (): void => {
+    setIsLoading(true)
+    window.location.href = `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/auth/github`
+  }
+
+  const handleGithubRedirect = async () => {
+    const urlParams = new URLSearchParams(window.location.search)
+    const code = urlParams.get('code')
+
+    if (code) {
+      try {
+        const response = await fetch(
+          `${process.env.NEXT_PUBLIC_BACKEND_URL}/api/auth/github/callback?code=${code}`,
```

Comment:
on this line, you are hitting `{{BASE_URL}}/api/auth/github/callback` but in the backend, we have got `{{BASE_URL}}/api/auth/github` only. Attaching an image for your reference. Do it this way and let us know if you still face any errors.

![Screenshot (492)](https://github.com/user-attachments/assets/ea91ad3b-bd74-42a0-8780-6f3a968aa385)
 here.

Reply:
Here's the flow @dev-palwar:
- On clicking the GitHub button, users should be redirected to `<API_BASE_URL>/api/auth/github`
- The API handles all the logic in here and then sends the user to github auth page.
- The user authenticates themselves on github
- Github sends back a redirect to the API with authentication success.
- The backend then sends back a redirect to a `<FRONTEND_BASE_URL>/oauth/signin` and attaches a `data` query object to this redirect url that has all the user data.